chapter_5/exercise_5_06/getline.c

```diff
@@ -1,23 +1,19 @@
 #include <stdio.h>
 
-void get_line(char *s);
 
-int main(void)
-{
-  char string[150] = "";
+char *getln(char *line);
 
-  get_line(string);
-  puts(string);
-
-  return 0;
+int main() {
+    char line[100];
+    
+    getln(line);
+    printf("%s", line);
+    
+    return 0;
 }
 
-void get_line(char *s)
-{
-  while ((*s = getchar()) != EOF && (*s != '\n'))
-  {
-    ++s;
-  }
-
-  *s = '\0';
+char *getln(char *line) {
```

Comment:
@kyopa For style consistently I will not move forward with getln. In fact, I don't prefer using short names for functions and variables. Since, the initial days of C/C++ the programming community shared many wise decisions, and short names seem to be already proven not wise. It's not bad, it simply doesn't scale in big applications, especially if the names are not part of the standard but are created by us. If you create a name that later will be used by many people, let's say more than 50, it'll be wise to make it very easy to understand. A short name could make sense though in a local context, like an iterator (most of the time in c people use `i` as a name), since a local variable it's not exposed to other people.